nerdlets/groundskeeper-v2/hooks/useFetchEntitiesDetails.js

```diff
@@ -189,10 +210,23 @@ const parseRuntimeType = (language, value) => {
   }
 };
 
-const featuresCheck = ({ attribute = '', value = '' }, feature = '') => {
-  if (attribute !== featuresList[feature] || !value) return false;
-  const val = value.toLowerCase();
-  return val !== 'false' && val !== 'none' && value !== '0';
-};
+  const featuresCheck = ({ attribute = '', value = '' }, feature = '') => {
+    const featureValue = featuresList[feature];
+  
+    if (Array.isArray(featureValue)) {
+      for (const item of featureValue) {
+        if (attribute === item && value) {
+          const val = value.toLowerCase();
+          return val !== 'false' && val !== 'none' && value !== '0';
+        }
+      }
+      return false;
+    } else {
+      if (attribute !== featureValue || !value) return false;
+      const val = value.toLowerCase();
+      return val !== 'false' && val !== 'none' && value !== '0';
```

Comment:
(see previous comment) This is similar to lines 219 and 220. I'd recommend turning this part into a function.